bpfman/src/bpf.rs

```diff
@@ -55,7 +55,7 @@ const MAPS_USED_BY_PREFIX: &str = "map_used_by_";
 pub(crate) struct BpfManager {
     config: Config,
     commands: Option<Receiver<Command>>,
-    image_manager: Option<Sender<ImageManagerCommand>>,
+    pub(crate) image_manager: Option<Sender<ImageManagerCommand>>,
```

Comment:
I might need an accessor here instead maybe?

Reply:
I'll be fixing that today/tomorrow. any of these "managers" just need to become plain old library functions now